src/management/apps.ts

```diff
@@ -2,11 +2,16 @@ import * as api from "../api";
 import * as FirebaseError from "../error";
 import * as logger from "../logger";
 import { pollOperation } from "../operation-poller";
+import { inspect } from "util";
 
 const TIMEOUT_MILLIS = 30000;
 const APP_LIST_PAGE_SIZE = 100;
 const CREATE_APP_API_REQUEST_TIMEOUT_MILLIS = 15000;
 
+const IOS_CONFIG_FILE_NAME = "GoogleService-Info.plist";
```

Comment:
I want to avoid hardcoding any of these strings in the CLI - The API response from Firebase should already include the preferred filename, so we can use that.

Reply:
I don't think our API returns `fileName` for web app, and we decided to name it `google-config.js`. I only removed default file name for iOS and Android apps for now

Reply:
LGTM, thanks.